### ISSUE TITLE
No audio in sideview with Manim Voiceover code ?

### ISSUE THREAD

**Issue by FredoT63** [Rickaym/manim-sideview#107]
Great extension, but manim-sideview seems to play video only. 
Maybe I've missed a configuration somewhere.

OS : Debian testing

**Comment by Rickaym** [comment]
Yes, as of now, the side view does not support audio. Tho this is planned to change in the near future.

**Comment by Rickaym** [comment]
This seems to be a limitation with VSCode. Video elements are forced to mute.

**Comment by FredoT63** [comment]
This is a sad news, but thank you for the answer.

**Comment by Rickaym** [comment]
No worries! I might try out alternative approaches like having a separate audio element synced to the voiceover--but it won't be the voice in the video.
